### ISSUE TITLE
typescript class jump to detention when there is constructor function

### ISSUE THREAD

**Issue by zivni** [Microsoft/vscode#23941]
- VSCode Version:1.10.2
- OS Version: windows 10

Steps to Reproduce:
```typescript
abstract class Base {
    constructor(){}
}
class Derived extends Base{}
let d=new Derived()
```
I don't know if it just me, but it really annoying that when you want to jump to  the class definition from a line where you create a new instance (like the line `let d=new Derived()` above) it jumps to the constructor function, and if the constructor function is defined in a parent class it jumps to the parent class and not to the class where the cursor was on, when you pressed F12.

I know that technically you call the constructor function and not the class but from navigation point of view - you want to go to the class.

**Comment by chrmarti** [comment]
You could argue both ways. Marking as feature request.

**Comment by mjbvz** [comment]
@zivni I believe you are looking for `Go to Type Definition`. In your example, this command will jump to  `class Derived`. This is already implemented and should work if you are using TS 2.1.5+

Please let me know if you run into any issues with this command

**Comment by zivni** [comment]
`Go to Type Definition` did the trick - thanks!
